### DISCUSSION TITLE
How can I keep changed playing speed for all new files?

### DISCUSSION THREAD

**Discussion by xxxDSSxxx** [smplayer-dev/smplayer#1011] [Q&A] [answered]
When I change playing speed it goes back to 1x for other files? How can I keep my speed for the current session (until I close the player) or forever (until I change it again) for all files?

**Comment by smplayer-dev** [comment]
Check the option "Keep selected speed across files" in Preferences -> General.

**Reply by xxxDSSxxx** [reply]
I do not have that option in my default version of SMPlayer in Ubuntu 2022. Can I somehow configure it in config file mb?
https://imgur.com/WWFNrby.png

**Reply by smplayer-dev** [reply, marked as answer]
That option was added in version 23.6.0.
In older versions you can add something like `--speed=1.2` in preferences -> advanced -> mpv -> options.

**Reply by xxxDSSxxx** [reply]
ok ty, I'll install the new version.
that string in options does not work for me, tried for both mvp and mplayer, and -speed (with one '-') also, got an error, or just playing without speed change
